openedx/core/djangoapps/coursegraph/management/commands/tests/utils.py

```diff
@@ -26,6 +26,26 @@ def begin(self):
         """
         return MockTransaction(self)
 
+    def commit(self, transaction):
```

Comment:
Between 3.1 and 2021.1, the methods `Transaction.commit()` and `Transaction.rollback()` were deprecated in favor of equivalent methods `Graph.commit(transaction)` and `Graph.rollback(transaction)`.